railties/lib/rails/mailers_controller.rb

```diff
@@ -5,8 +5,9 @@
 class Rails::MailersController < Rails::ApplicationController # :nodoc:
   prepend_view_path ActionDispatch::DebugView::RESCUES_TEMPLATE_PATH
 
+  around_action :set_locale, only: :preview
+  before_action :find_preview, only: :preview
   before_action :require_local!, unless: :show_previews?
```

Comment:
Since this is gated by the route to simply not be present, I'm not certain why it is required. didn't alter it as part of the PR it could be done on the side if we desire.